ckanext/unaids/theme/templates/request/page.html

```diff
@@ -0,0 +1,2 @@
+{% extends "page.html" %}
```

Comment:
Why do you need this template here? You're not introducing any changes to it compared to ytp-request.